server/controllers/admin.ts

```diff
@@ -124,73 +130,91 @@ const controllers: IControllerAdmin = {
   },
 
   async blockCommentThread(
-    ctx: StrapiRequestContext<never>
+    ctx: StrapiRequestContext<never>,
   ): ThrowablePromisedResponse<Comment> {
-    const { params = {} } = ctx;
-    const { id } = parseParams(params);
+    const {params = {}} = ctx;
+    const {id} = parseParams(params);
     try {
       return await this.getService<IServiceAdmin>().blockCommentThread(
         id,
-        true
+        true,
       );
     } catch (e) {
       throw throwError(ctx, e);
     }
   },
 
   async unblockCommentThread(
-    ctx: StrapiRequestContext<never>
+    ctx: StrapiRequestContext<never>,
   ): ThrowablePromisedResponse<Comment> {
-    const { params = {} } = ctx;
-    const { id } = parseParams(params);
+    const {params = {}} = ctx;
+    const {id} = parseParams(params);
     try {
       return await this.getService<IServiceAdmin>().blockCommentThread(
         id,
-        false
+        false,
       );
     } catch (e) {
       throw throwError(ctx, e);
     }
   },
 
   async resolveAbuseReport(
-    ctx: StrapiRequestContext<never>
+    ctx: StrapiRequestContext<never>,
   ): ThrowablePromisedResponse<CommentReport> {
-    const { params = {} } = ctx;
-    const { id: commentId, reportId } = parseParams(params);
+    const {params = {}} = ctx;
+    const {id: commentId, reportId} = parseParams(params);
     try {
       return await this.getService<IServiceAdmin>().resolveAbuseReport(
         reportId,
-        commentId
+        commentId,
+      );
+    } catch (e) {
+      throw throwError(ctx, e);
+    }
+  },
+
+  async resolveCommentMultipleAbuseReports(
+    ctx: StrapiRequestContext<never>,
+  ): ThrowablePromisedResponse<StrapiDBBulkActionResponse> {
+    const {params = {}, request} = ctx;
+    const {body} = request;
+    const {id: commentId} = parseParams(params);
+
+    assertNotEmpty<Array<Id>>(body);
+
+    try {
+      return await this.getService<IServiceAdmin>().resolveCommentMultipleAbuseReports(
```

Comment:
```suggestion
      return await this.getService<IServiceAdmin>().resolveMultipleAbuseReports(
```
IMHO it can be more concise

Reply:
There is another method with the same name as suggested change, that works with different context